### DISCUSSION TITLE
Data Importer: the process does not move

### DISCUSSION THREAD

**Discussion by r-set** [pimcore/pimcore#12143] [Q&A / Support] [answered]
Hi,

Pimcore X.
I'm trying to import data from csv. After marking and starting the import, the process does not move.
Do you have any idea what could be the problem here?
![image](https://user-images.githubusercontent.com/96417079/168889923-9bc47d13-53be-42e9-bd55-abbded4b65a7.png)
![image](https://user-images.githubusercontent.com/96417079/168887595-6c06ec78-c76f-48f5-92de-037c1fb6946b.png)
![image](https://user-images.githubusercontent.com/96417079/168887808-cdb400cb-7080-4c73-bc4e-543d7affaecc.png)

**Comment by mcop1** [comment, marked as answer]
@r-set Please take a look at 
https://pimcore.com/docs/data-importer/current/Installation.html - I think you are either missing the command or you need to activate the symfony messenger for data-importer.

Hope this helps, 
best regards

**Reply by r-set** [reply]
@mcop1 Thanks! I'll try to install again, as there may be a problem with file permissions.
